stable/datacube-index/templates/cronjob-index.yaml

```diff
@@ -0,0 +1,109 @@
+{{- if .Values.cron.enabled }}
+{{- $externalDb := .Values.global.externalDatabase }}
+{{- $dc := .Values.global.datacube }}
+{{- $index := .Values.index }}
+{{- $image := .Values.global.image }}
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: "{{ $index.jobname }}"
```

Comment:
Should probably define a default value for this in `values.yaml` and possibly move it to `.values.cron.jobname`